docs/Contributing/API-for-contributors.md

```diff
@@ -527,63 +527,158 @@ Delete pack by name.
 
 The MDM endpoints exist to support the related command-line interface sub-commands of `fleetctl`, such as `fleetctl generate mdm-apple` and `fleetctl get mdm-apple`, as well as the Fleet UI.
 
-- [Generate Apple DEP Key Pair](#generate-apple-dep-key-pair)
+- [Generate Apple Business Manager public key (ADE)](#generate-apple-business-manager-public-key-ade)
 - [Request Certificate Signing Request (CSR)](#request-certificate-signing-request-csr)
+- [Upload APNS certificate](#upload-apns-certificate)
+- [Upload ABM Token](#upload-abm-token)
+- [Turn off Apple MDM](#turn-off-apple-mdm)
+- [Disable automatic enrollment (ADE)](#disable-automatic-enrollment-ade)
 - [Batch-apply MDM custom settings](#batch-apply-mdm-custom-settings)
 - [Initiate SSO during DEP enrollment](#initiate-sso-during-dep-enrollment)
 - [Complete SSO during DEP enrollment](#complete-sso-during-dep-enrollment)
 - [Preassign profiles to devices](#preassign-profiles-to-devices)
 - [Match preassigned profiles](#match-preassigned-profiles)
 - [Get FileVault statistics](#get-filevault-statistics)
 
-### Generate Apple DEP Key Pair
+### Generate Apple Business Manager public key (ADE)
 
-#### Parameters
+`GET /api/v1/fleet/mdm/apple/abm_public_key`
 
-None.
+#### Example
+
+`GET /api/v1/fleet/mdm/apple/abm_public_key`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+    "public_key": "23K9LCBGG26gc2AjcmV9Kz="
+}
+```
+
+### Request Certificate Signing Request (CSR)
+
+`GET /api/v1/fleet/mdm/apple/request_csr`
 
 #### Example
 
-`POST /api/v1/fleet/mdm/apple/dep/key_pair`
+`GET /api/v1/fleet/mdm/apple/request_csr`
```

Comment:
If we move this to the public API at some point, I'd suggest renaming this to `GET /api/v1/fleet/mdm/apple/csr` (the "request" feels a little redundant)